### PR TITLE
Add dashboard url to module api results

### LINKED ISSUE #2596: Cypress module API should return dashboard URL of recorded run
### Current behavior:

I'm running Cypress programmatically with the module API. I want to send a Slack message with the dashboard URL for failed runs. Currently, the dashboard URL of the recorded run is printed to stdout but not included in the return value of `cypress.run`.

### Desired behavior:

`cypress.run` return value should include a key called `dashboardUrl` with the a value like `https://dashboard.cypress.io/#/projects/asdf/runs/123`.

### Versions

Cypress 3.0.2

### PR DESCRIPTION
**Pull request by lilaconlee**
- Fixes #2596

### CODE REVIEW

**Comment by brian-mann** [comment]
Let's rename this property to be `runUrl`. It matches what our API sends and helps to clarify the destination. `dashboardUrl` could be referring to any URL on the dashboard. For instance, we could add things like `projectUrl` or `organizationUrl` if needed to go alongside it.

File: packages/server/test/e2e/7_record_spec.coffee [hunk 1]
```diff
@@ -304,6 +311,10 @@ describe "e2e record", ->
         expect(forthInstanceStdout.body.stdout).not.to.include("record_fail_spec.coffee")
         expect(forthInstanceStdout.body.stdout).not.to.include("record_pass_spec.coffee")
 
+        fs.readJsonAsync(outputPath)
+        .then (results) ->
+          expect(results.runUrrunUrl).to.equal(runUrl)
```
Comment (brian-mann) [inline packages/server/test/e2e/7_record_spec.coffee]: `runUrrunUrl` made me lol

Comment (lilaconlee) [reply packages/server/test/e2e/7_record_spec.coffee]: a perfectly legitimate key name

File: packages/server/lib/modes/run.coffee [hunk 2]
```diff
@@ -767,6 +767,7 @@ module.exports = {
       osName: sys.osName,
       osVersion: sys.osVersion,
       cypressVersion: pkg.version,
+      runUrl: runUrl,
```
Comment (brian-mann) [inline packages/server/lib/modes/run.coffee]: Super minor, not worth fixing (will merge), but you can omit values when they match the key names... now that I think about this why isn't this a lint rule... oh wait we're still on coffeescript, oh nevermind then.

**Pull request merged**